src/mesh/PhoneAPI.cpp

```diff
@@ -340,13 +345,28 @@ size_t PhoneAPI::getFromRadio(uint8_t *buf)
             nodeInfoForPhone.num = 0; // We just consumed a nodeinfo, will need a new one next time
         } else {
             LOG_INFO("Done sending nodeinfos\n");
-            state = STATE_SEND_COMPLETE_ID;
+            state = STATE_SEND_FILEMANIFEST;
             // Go ahead and send that ID right now
             return getFromRadio(buf);
         }
         break;
     }
 
+    case STATE_SEND_FILEMANIFEST: {
+        LOG_INFO("getFromRadio=STATE_SEND_FILEMANIFEST\n");
+        fromRadioScratch.which_payload_variant = meshtastic_FromRadio_fileInfo_tag;
+        if (config_state < filesManifest.size()) {
```

Comment:
Has this ever been tested? What if size() returns 0? config_state is **unsigned**, can never be < 0

Reply:
Received: { "fileInfo": { "fileName": "/prefs/channels.proto", "sizeBytes": 57 } }
Received: { "fileInfo": { "fileName": "/prefs/config.proto", "sizeBytes": 98 } }
Received: { "fileInfo": { "fileName": "/prefs/db.proto", "sizeBytes": 700 } }
Received: { "fileInfo": { "fileName": "/prefs/module.proto", "sizeBytes": 104 } }
Received: { "fileInfo": { "fileName": "/static/.gitkeep" } }
Received: { "fileInfo": { "fileName": "/static/Logo_Black.svg.gz", "sizeBytes": 602 } }
Received: { "fileInfo": { "fileName": "/static/Logo_White.svg.gz", "sizeBytes": 610 } }
Received: { "fileInfo": { "fileName": "/static/apple-touch-icon.png.gz", "sizeBytes": 3164 } }
Received: { "fileInfo": { "fileName": "/static/favicon.ico.gz", "sizeBytes": 2270 } }
Received: { "fileInfo": { "fileName": "/static/icon.svg.gz", "sizeBytes": 852 } }
Received: { "fileInfo": { "fileName": "/static/index-B272zAn3.css.gz", "sizeBytes": 16427 } }
Received: { "fileInfo": { "fileName": "/static/index-CGqDWSD_.js.gz", "sizeBytes": 348 } }
Received: { "fileInfo": { "fileName": "/static/index-CgmZQ_N-.js.gz", "sizeBytes": 204401 } }
Received: { "fileInfo": { "fileName": "/static/index.html.gz", "sizeBytes": 549 } }
Received: { "fileInfo": { "fileName": "/static/maplibre-gl-69L4OEQ1.js.gz", "sizeBytes": 210430 } }
Received: { "fileInfo": { "fileName": "/static/robots.txt.gz", "sizeBytes": 42 } }
Received: { "fileInfo": { "fileName": "/static/site.webmanifest.gz", "sizeBytes": 197 } }
Received: { "configCompleteId": 1567494388 }